.github/workflows/docs.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     tags:
       - "*"
+  workflow_dispatch:
```

Comment:
cf. https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch